vpc/tool/bpfloader/bpfloader_linux_test.go

```diff
@@ -14,6 +16,15 @@ func TestLoad(t *testing.T) {
 	if os.Getenv("CIRCLECI") == "true" {
 		t.Skip("Test does not work on Travis")
 	}
+
+	content, err := ioutil.ReadFile("/proc/sys/kernel/unprivileged_bpf_disabled")
+	assert.NoError(t, err)
+	u, err := user.Current()
+	assert.NoError(t, err)
+	if string(content) != "0\n" && u.Uid != "0" {
```

Comment:
strconv.ParseInt?

Reply:
Sure, although then we have to check another error. I can make the change on Monday.